gulpfile.js

```diff
@@ -27,8 +27,8 @@ gulp.task('sass-bootstrap', ['sass-sync'], function () {
     .pipe(gulp.dest('css/'));
 });
 
-gulp.task('sass-civicrm', ['sass-sync'], function () {
-  gulp.src('scss/civicrm/custom-civicrm.scss')
+gulp.task('sass:civicrm', ['sass:sync'], function () {
+  return gulp.src('scss/civicrm/custom-civicrm.scss')
```

Comment:
Ah, nice. So this is probably why `gulp` was hanging for a moment before finishing?

https://github.com/gulpjs/gulp/blob/master/docs/API.md#async-task-support